raytracing/figure.py

```diff
@@ -467,12 +545,20 @@ def create(self, comments=None, title=None):
 
         self.axes.set(xlabel='Distance', ylabel='Height', title=title)
 
-    def display2D(self, filepath=None):
+    def display2D(self, interactive=True, filepath=None):
         self.draw()
 
         self.axes.callbacks.connect('ylim_changed', self.onZoomCallback)
         plt.connect('resize_event', self.onZoomCallback)
 
+        if interactive:
+            visibility = self.visibility
+            visibility.pop('elements')
+
+            subAxes = plt.axes([0.91, 0.65, 0.08, 0.05*len(visibility)])
```

Comment:
The position is awkward.  I would put it probably higher and use larger fonts.
Also, *no box* around!

Reply:
Also, in the `__main__`, the checkbox is half the size and looks squished.

Reply:
Is this ok ? 
![image](https://user-images.githubusercontent.com/29587649/87820705-8bf67800-c83c-11ea-9879-2b2ebac5f99d.png)

![image](https://user-images.githubusercontent.com/29587649/87820735-9b75c100-c83c-11ea-9e3e-ec50959a9f4d.png)